configure.ac

```diff
@@ -115,7 +115,7 @@ AC_SUBST([HARDEN_CFLAGS])
 AC_SUBST([HARDEN_LDFLAGS])
 
 # Also check for a few non-hardening-related flags.
-MISC_CXXFLAGS=""
+MISC_CXXFLAGS="-std=c++11"
```

Comment:
FWIW, this PR fails to build on my fairly ancient moshing machine at work:

```
cc1plus: error: unrecognized command line option "-std=c++11"
```

I've for sure appreciated how portable mosh is to just about every linux environment I've been on.